costum_events/views/myevents.xml

```diff
@@ -28,33 +25,25 @@
               </group>
             </page>
           </xpath>
-        
         </field>       
   </record> 
 <!-- event.event tree view -->
-
   <record id="name_view_tree" model="ir.ui.view">
```

Comment:
this wrong place to inherit you should inherit the form view only and it should be in the same record